operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -0,0 +1,342 @@
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.logging.Logger;
+
+import javax.ws.rs.core.MultivaluedHashMap;
+import javax.ws.rs.core.MultivaluedMap;
+
+import com.ibm.fhir.exception.FHIROperationException;
+import com.ibm.fhir.model.format.Format;
+import com.ibm.fhir.model.parser.FHIRParser;
+import com.ibm.fhir.model.parser.exception.FHIRParserException;
+import com.ibm.fhir.model.resource.Bundle;
+import com.ibm.fhir.model.resource.Bundle.Entry;
+import com.ibm.fhir.model.resource.OperationDefinition;
+import com.ibm.fhir.model.resource.Parameters;
+import com.ibm.fhir.model.resource.Parameters.Parameter;
+import com.ibm.fhir.model.resource.Patient;
+import com.ibm.fhir.model.resource.Resource;
+import com.ibm.fhir.model.type.UnsignedInt;
+import com.ibm.fhir.model.type.Uri;
+import com.ibm.fhir.model.type.code.BundleType;
+import com.ibm.fhir.model.type.code.IssueType;
+import com.ibm.fhir.search.compartment.CompartmentUtil;
+import com.ibm.fhir.search.exception.FHIRSearchException;
+import com.ibm.fhir.server.operation.spi.AbstractOperation;
+import com.ibm.fhir.server.operation.spi.FHIROperationContext;
+import com.ibm.fhir.server.operation.spi.FHIRResourceHelpers;
+import com.ibm.fhir.server.util.FHIROperationUtil;
+
+
+/**
+ * This class implements the <a href="https://www.hl7.org/fhir/operation-patient-everything.html">$everything</a> operation
+ * which is used to return all the information related to one or more patients described in the resource or context on 
+ * which this operation is invoked.
+ * 
+ * @author Luis A. García
+ */
+public class EverythingOperation extends AbstractOperation {
+
+    private static final Logger LOG = java.util.logging.Logger.getLogger(EverythingOperation.class.getName());
+    
+    /**
+     * The <a href="https://www.hl7.org/fhir/search.html#prefix">prefix</a> used to indicate the start date for the $everything resources
+     */
+    protected static final String STARTING_FROM = "ge";
+
+    /**
+     * The <a href="https://www.hl7.org/fhir/search.html#prefix">prefix</a> used to indicate the end date for the $everything resources
+     */
+    protected static final String UP_UNTIL = "le";
+
+    /**
+     * The "date" query parameter used in the underlying search operation.
+     */
+    protected static final String DATE_QUERY_PARAMETER = "date";
+
+    /**
+     * The "_lastUpdated" query parameter used in the underlying search operation.
+     */
+    protected static final String LAST_UPDATED_QUERY_PARAMETER = "_lastUpdated";
+
+    /**
+     * The query parameter to indicate a maximum resource count for the $everything operation
+     */
+    protected static final String COUNT_QUERY_PARAMETER = "_count";
+
+    /**
+     * The query parameter to indicate the resource types for the $everything operation
+     */
+    protected static final String TYPE_QUERY_PARAMETER = "_type";
+
```

Comment:
should deduplicate with SearchConstants

Reply:
I didn't see a specific constant for `_lastUpdated` but I added one.
As far as _count and _type, done.